packaging/debian-sid/changelog

```diff
@@ -1,3 +1,234 @@
+    - tests: update os.query check to match new bullseye codename used
+      on sid images
+    - o/snapstate: helper for getting snaps affected by refresh, define
+      new hook
+    - wrappers: support in EnsureSnapServices a callback to observe
+      changes (#10176)
+    - gadget: multi line support in gadget's cmdline file
+    - daemon: test that requesting restart from (early) Ensure works
+    - overlord: make servicestate responsible to compute
+      SnapServiceOptions (#10171)
+    - tests: use op.paths tools instead of dirs.sh helper - part 1
+    - tests: add new command to snaps-state to get current core, kernel
+      and gadget
+    - boot, gadget: move opening the snap container into the gadget
+      helper
+    - tests, overlord: extend unit tests, extend spread tests to cover
+      full command line support
+    - interfaces/builtin: introduce dsp interface
+    - boot, bootloader, bootloader/assets: support for full command line
+      override from gadget
+    - overlord/devicestate, overlord/snapstate: add task for updating
+      kernel command lines from gadget
+    - o/snapstate: remove unused DeviceCtx argument of
+      ensureInstallPreconditions
+    - tests/lib/nested: proper status return for tpm/secure boot checks
+    - cmd/snap, boot: add snapd_full_cmdline_args to dumped boot vars
+    - wrappers/services.go: refactor helper lambda function to separate
+      function
+    - boot/flags.go: add HostUbuntuDataForMode
+    - boot: handle updating of components that contribute to kernel
+      command line
+    - tests: add 20.04 to systems for nested/core
+    - wrappers/services.go: introduce EnsureSnapServices()
+    - daemon: add new accessChecker implementations
+    - boot, overlord/devicestate: consider gadget command lines when
+      updating boot config
+    - tests: fix prepare-image-grub-core18 for arm devices
+    - tests: fix gadget-kernel-refs-update-pc test on arm and when
+      $TRUST_TEST_KEY is false
+    - tests: enable help test for all the systems
+    - boot: set extra command line arguments when preparing run mode
+    - boot: load bits of kernel command line from gadget snaps
+    - tests: update layout for tests - part 2
+    - tests: update layout for tests - part 1
+    - tests: remove the snap profiler from the test suite
+    - boot: drop gadget snap yaml which is already defined elsewhere in
+      the tests
+    - boot: set extra kernel command line arguments when making a
+      recovery system bootable
+    - boot: pass gadget path to command line helpers, load gadget from
+      seed
+    - tests: new os.paths tool
+    - daemon: make ucrednetGet() return a *ucrednet structure
+    - boot: derive boot variables for kernel command lines
+    - boot,tests: move where we write boot-flags one level up*
+      boot,tests: move where we write boot-flags one level up* cmd/snap-
+      bootstrap/initramfs-mounts: fix boot-flags location from initramfs
```

Comment:
something is a bit weird about this one too but is hard to parse what

Reply:
I split this one up/fixed it, some of the things in this were actually also in the 2.50 changelog too